indico/modules/events/registration/models/items.py

```diff
@@ -437,6 +437,10 @@ def view_data(self):
     def _get_default_log_data(self):
         return {'Section ID': self.id}
 
+    @make_interceptable
+    def is_visible(self, is_management):
+        return not self.is_deleted and (is_management or not self.is_manager_only)
```

Comment:
Doesn't this change the logic? Previously, deleted sections would still be visible in management, but that does not seem to be the case here

Reply:
Sorry, original logic restored.